string mission_name
string pycode
---
bool success
string message